back/engines/commercial/analysis/app/lib/analysis/auto_tagging_method/base.rb

```diff
@@ -160,6 +160,21 @@ def deduct_locale(input)
         AppConfiguration.instance.settings('core', 'locales').first
     end
 
+    def classification_fields
+      fields = analysis.associated_custom_fields
```

Comment:
It is not clear what's in the `associated_custom_fields` and what's not. This method is adding the `topic_ids` custom field if it's missing. However, in which case is it already present or not?

Reply:
Currently, topic IDs are never included in the associated custom fields. These are set through the frontend to: the title, the body and the location (so this behaviour could potentially change).